app/Resources/views/search/search.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
```

Comment:
If this is a base this should be named `app/Resources/views/search/layout.html.twig`